### PR TITLE
Support for the XDG Base Directory Specification.

### PR DESCRIPTION
**Pull request by denisfa**
Hello again @dave-theunsub. 

I finally had some free time to do and review some coding. Here we have the XDG support that I wanted in #93 . I think I got everything right, but feel free to comment whatever you want. I am a python dev, so some stuff for me took some time to learn.

BTW, do you have in perl something like pip from python? It would be nice if we had a deps list to let maintainers know what they need to install in order to use. In this, for example, I added the "File::BaseDir" lib.

[]s,
Edênis

### CODE REVIEW

**Comment by dave-theunsub** [comment]
Hi Edênis,

Thank you for your work.  It looks good from here.  However, my development system died... I'm hoping to get another soon.  Please be patient while I figure out how to do all that. :)

Perl does have cpan to install packages, but with any open source program I write, I always ensure the requirements are met with the system's package manager (apt-get, yum, dnf, etc).  One of the biggest goals, especially with ClamTk, was that anyone could point and click and install.

Having said all that, I'm pretty sure File::BaseDir is a built-in, so it shouldn't be a problem.  I'll be able to figure out for sure soon.

Long story short - I really like what you did here.  Thanks again

respectfully
Dave M

**Comment by denisfa** [comment]
Hey @dave-theunsub ! Take your time!

I did some stupid editing on my repo and ended up being what you are seeing above. Ignore these. I wanted to make another pull request for another stuff but ended up with this.

Sorry!

[]s,
Edênis
